### DISCUSSION TITLE
"vim/shared.lua:0: after the second argument: expected table, got string" whenever call lazygit out

### DISCUSSION THREAD

**Discussion by davidgao7** [LazyVim/LazyVim#3672] [Q&A]
Hello everyone,

I got this message everytime I call lazygit out, it happened after I synced plugins this morning.

<img width="678" alt="Screenshot 2024-06-15 at 9 50 48 AM" src="https://github.com/LazyVim/LazyVim/assets/39502459/c4386fc4-53be-4b80-8ad9-bd275540958f">

the first layer of Which-key seems to be overwritten

<img width="878" alt="Screenshot 2024-06-15 at 10 11 28 AM" src="https://github.com/LazyVim/LazyVim/assets/39502459/8bf0bfb9-75dc-4d19-9495-49b1d7f95136">

**Comment by dotfrag** [comment]
Do you get the same behaviour when using `repro.lua`?

**Reply by davidgao7** [reply]
Hello @dotfrag, thx for replying,
I haven't used/created/modified `repro.lua`,   or I probably don't get what you saying sorry

the highlight is acting weird too
<img width="708" alt="Screenshot 2024-06-15 at 2 21 20 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/4f7b52d4-7b48-43ce-b43c-9a339f75d835">
<img width="
<img width="498" alt="Screenshot 2024-06-15 at 4 49 29 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/5ce5287c-386f-47fe-b024-dfcf9bbb56df">
<img width="305" alt="Screenshot 2024-06-15 at 4 50 12 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/0d90bded-8488-4b2d-b13a-03289c69d2fd">
<img width="548" alt="Screenshot 2024-06-15 at 2 22 37 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/3fbfb2ca-873c-43b7-a24d-1a5d839e3615">

**Reply by davidgao7** [reply]
every buffer looks terrifying... 
<img width="661" alt="Screenshot 2024-06-15 at 5 10 34 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/aac97123-4c3f-4114-86fd-c9fa5a155bca">

**Reply by dpetka2001** [reply]
About the first error in your OP unfortunately you're not giving enough info as to what might be going on. As for the colors, usual culprits are the colorscheme you use and the terminal emulator you use (terminal support for true colors and tmux proper configuration if you use it come to mind).

**Reply by davidgao7** [reply]
ok omzsh just updated... lazygit is no longer complaining, everything goes back to normal, sorry
really don't know what's going on... very curious
change log doesn't mention any 'color' change
<img width="859" alt="Screenshot 2024-06-15 at 11 13 59 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/b228610e-bc9d-4b2d-a63f-7dbd09c04db5">

I wonder anyone who might know why it behaves like this could tell me what's going on, should I keep the discussion open? can people comment a closed discussion?
